docs/user_manual/dataset-terminology.md

```diff
@@ -10,23 +10,101 @@ Some terms regarding the data structures are explained here, including the defin
+
+### Dimensions of numpy arrays
+
+The dimensions of numpy arrays and the interpretation of each dimension is as follows.
+
+| **Data Type**            | **1D**                                                                                       | **2D**                                                                                                           | **3D**                                                                                                           |
+|--------------------------|----------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| **SingleArray**          | Corresponds to a single dataset.                                                             | ❌                                                                                                               | ❌                                                                                                               |
+| **DenseBatchArray**      | ❌                                                                                            | Batch number $\times$ Component within that batch                                                                       | ❌                                                                                                               |
+| **SingleColumn**   | Component within that batch.                                                                 | Component within that batch $\times$ Phases &#10024;                                                                            | ❌                                                                                                               |
+| **BatchColumn**    | ❌                                                                                            | Batch number $\times$ Component within that batch                                                                       | Batch number $\times$ Component within that batch $\times$ Phases &#10024;                                                            |
+
+```{note}
+&#10024; The "Phases" dimension is optional and is available only when the attributes are asymmetric.
+```
+
+### Type of Dataset
 
 The types of `Dataset` include the following: `input`, `update`, `sym_output`, `asym_output`, and `sc_output`:
 Exemplery datasets attributes are given in a dataset containing a `line` component.
 
-- **input:** Contains attributes relevant to configuration of grid. 
+- **input:** Contains attributes relevant to configuration of grid.
   - Example: `id`, `from_node`, `from_status`
-- **update:** Contains attributes relevant to multiple scenarios. 
+- **update:** Contains attributes relevant to multiple scenarios.
```

Comment:
here and earlier: maybe nice to explicitly provide links to the `DatasetType` enum